### ISSUE TITLE
A screen reader does not message that the button is a button

### ISSUE THREAD

**Issue by kranid** [kranid/Telegram#1]
[Talkback](https://support.google.com/accessibility/android/answer/6283677?hl=en) does not announce some buttons as a button. It may confuses  the talkback users and complicate the interaction with the application for them.

Steps to reproduce:
1. [Activate Talkback](https://support.google.com/accessibility/android/answer/6007100);
2. launch telegram application;
3. Go to the intro screen (IntroActivity);
4. focus on the Start Messaging button;
5. focus on the Continue in native language button;

Actual result: Talkback  speaks only the label of the button but does not message that this control is  button:

- "Start messaging"
- "Continue in native language"

Expected result: talkback announces these controls as follows:

- "Start messaging button"
- "Continue in native language button"
